library/aws/ec2-deployment.ts

```diff
@@ -5,7 +5,8 @@ import * as AR from '../../providers/aws/resources.ts';
 
 import * as aws from './aws.ts';
 import * as roles from './roles.ts';
-import * as policies from './policies.ts';
+import * as policies from './policies_v2.ts';
```

Comment:
why import `*` but then import specific functions from the same file? (line 19)

Reply:
same in `autoscale-deployments.ts`

Reply:
Because we use the specific functions many times so want a shorter, non qualified, name, whilst we use many other functions only once, and don't want to have a massive list of explicit imports.